Resources/translations/SonataMediaBundle.en.xliff

```diff
@@ -78,6 +78,10 @@
                 <source>description.forbidden_download_strategy</source>
                 <target>The media can be retrieved by nobody.</target>
             </trans-unit>
+            <trans-unit id="description.session_download_strategy">
+                <source>description.session_download_strategy</source>
+                <target>The media can be retrieved %times% time(s) per session</target>
```

Comment:
Also, can you use [pluralization](http://symfony.com/doc/current/components/translation/usage.html#pluralization) ?